.rubocop.yml

```diff
@@ -27,6 +27,11 @@ Metrics/LineLength:
     - 'lib/moab/file_manifestation.rb'
     - 'spec/unit_tests/moab/bagger_spec.rb' # remove after PR #52 merged
     - 'spec/unit_tests/moab/file_group_difference_spec.rb' # remove after PR #50 merged
+    - 'spec/unit_tests/moab/storage_object_validator_spec.rb'
+
+# Max was 8
```

Comment:
think of these comments more as "why" than "what".  Like "code was readable with higher complexity allowed"